README.md

```diff
@@ -6,80 +6,4 @@
 
 PyDAGMC is a Python interface for interacting with DAGMC .h5m files through the embedded topological relationships and metadata contained within. These interactions occur through a set of Python classes corresponding to DAGMC’s metadata Group (a grouping of volumes or surfaces), Volume, and Surface groupings in the mesh database. This interface is intended to provide a simple interface for obtaining information about DAGMC models, replacing significant boilerplate code required to perform the same queries with PyMOAB, the Python interface for MOAB itself.
 
-PyDAGMC classes provide the ability to perform basic queries as properties of the class instances. These queries include:
```

Comment:
I think many users like to see some basic documentation like this in the README on GH so they don't have to dive into a complete documentation page.  I think it would be good to leave this here, but move the link to detailed documentation before this.